### PR TITLE
Made a script that parses run.json files and outputs the scores of each experiment in a results directory

### LINKED ISSUE #135: Automate report of scores from multiple runs
This would be a script that would output a csv per problem with the best scores of each model. I think there is use for two input options: 1) a results directory and 2) a newline separated file of results paths.

### PR DESCRIPTION
**Pull request by erikhuck**
closes #135

### CODE REVIEW

**Comment by bjschoenfeld** [comment]
You are missing option 2 from #135. Essentially what I want is a script that will take a csv like our run log in the google sheet and output the three separate leaderboards we have in that sheet, just in csv format.

**Comment by erikhuck** [comment]
> You are missing option 2 from #135. Essentially what I want is a script that will take a csv like our run log in the google sheet and output the three separate leaderboards we have in that sheet, just in csv format.

@bjschoenfeld I don't know what you mean by csv format. I understand that pandas can load a csv file. Would you like me to create a data frame from a file and print out particular columns or rows of a table?

**Comment by bjschoenfeld** [comment]
Thank you for updating the inputs. The output still needs some work. The output should be the same, independent of the input. I want 3 csv files as the output, not writing to std out. The format of those csv files should match the format of the pages in our tuning google sheet.

File: automatic_score_report.py [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+    total_rmse_scores = []
+    total_rmse_path_indices = []
+    mean_spearman_scores = []
+    mean_spearman_path_indices = []
+    top_k_regret_scores = []
+    top_k_regret_path_indices = []
+
+    for path_idx, results_dir in enumerate(paths):
+        results = load_results(results_dir)
+        problem_scores = results['scores']
```
Comment (bjschoenfeld) [inline automatic_score_report.py]: some results don't have scores if the experiment was terminated early or failed. this should handle that case

**Comment by bjschoenfeld** [comment]
@erikhuck A [csv file](https://en.wikipedia.org/wiki/Comma-separated_values) is just like a table, where all the values are separated by commas or newlines. You can use pandas if its easier. Let's make a table like those in the google sheet and write it to a `<some name>.csv` file. Does this help?

**Pull request merged**